Dockerfile.dev

```diff
@@ -0,0 +1,30 @@
+FROM node:18.15.0-bullseye-slim
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+  bash \
+  build-essential \
+  git \
+  unzip \
+  wget 
+
+ENV WATCHMAN_VERSION=v2024.02.05.00
```

Comment:
We may want to set the version dynamically and get the most recent release as watchman is released every week, now.